README.md

```diff
@@ -52,6 +52,26 @@ $ docker run \
   -p "4000:4000" starefossen/github-pages
 ```
 
+## Docker compose
+
+If you want to use Docker Compose instead, you can move a lot of the options into a `docker-compose.yml` at the root of your project:
+
+```
+version: '3'
+services:
+  jekyll:
+    image: starefossen/github-pages
+    environment:
+      - "JEKYLL_GITHUB_TOKEN:${JEKYLL_GITHUB_TOKEN}"
+    ports:
+      - "4000"
```

Comment:
Shouldn't it be "4000:4000"? With "4000" it is not working for me.

Reply:
I could've sworn it did work for me (I tested this numerous times), but now it doesn't.

Reply:
```suggestion
      - "4000:4000"
```